packages/button/src/Button.ts

```diff
@@ -61,10 +62,34 @@ export class Button extends SizedMixin(ButtonBase, { noDefaultSize: true }) {
     @property({ type: Boolean, reflect: true, attribute: true })
     public pending = false;
 
+    protected pendingStateController: PendingStateController<this>;
+
+    constructor() {
+        super();
+        this.pendingStateController = new PendingStateController(this, {
+            pending: () => this.pending,
+            onPendingChange: (isPending: boolean) => {
```

Comment:
Please create a documentation block on this methos!

Reply:
Done!